docs/ios/native-dependencies.md

```diff
@@ -0,0 +1,39 @@
+# Native dependencies
+
+## Add a new external native dependency
+
+All iOS code is kept in `ios` folder and using the same `Podfile` that's why process of adding a new external native dependency can be achieved by running a command:
+
+```bash
+react-native link NATIVE_DEPENDENCY_NAME
+```
+
+## Add a new internal native dependecy
```

Comment:
This one is about adding external native dependencies and internal packages too.